.husky/pre-commit

```diff
@@ -2,3 +2,4 @@
 . "$(dirname "$0")/_/husky.sh"
 
 pnpm exec lint-staged
+pnpm exec svelte-check --tsconfig ./tsconfig.json --fail-on-warnings --use-new-transformation
```

Comment:
This feels like such an expensive/time-consuming thing to run on every commit, since it checks the entire repo every time. How about we move it to `pre-push` instead?

Reply:
It's catching TypeScript errors, I think it's important to run before every commit. Also, it's not that time-consuming.